test/linarith.lean

```diff
@@ -188,8 +186,8 @@ example (i n : ℕ) (h : (2:ℤ) ^ i ≤ 2 ^ n) : (0:ℤ) ≤ 2 ^ n - 2 ^ i := b
 example (a b c : Rat) (h2 : b > 0) (h3 : b < 0) : Nat.prime 10 := by
   linarith
 
-example (a b c : Rat) (h2 : (2 : Rat) > 3)  : a + b - c ≥ 3 :=
```

Comment:
I would leave this, it doubles a test that `linarith` works on `>`

Reply:
Oh, indeed! Reverted.